.github/dependabot.yml

```diff
@@ -9,3 +9,12 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "daily"
+    ignore:
+      - dependency-name: "requests"
+      - dependency-name: "httpx"
```

Comment:
I think we should have notifications for these?

Reply:
Sure! The idea was to blanket all as the minimum versions we allow in `setup.cfg` but you're right that we definitely need updates on `httpx`'s status. `requests` can be dropped once the `v3-in-v4` client is removed also